### PR TITLE
Validate RBAC roles and bindings generated by tenant reconciler

### PR DESCRIPTION
**Pull request by shivi28**
1. start tenant and tenantnamespace controller
2. create tenant admin and nonadmin ser
3. validate tenantadmin is able to create tenantnamespace, whereas nonadmin user is not

### CODE REVIEW

File: tenant/pkg/controller/tenantnamespace/tenantnamespace_helper.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- context:
+    cluster: {{ .cluster }}
+    user: {{ .username }}
+  name: default
+current-context: default
+preferences: {}
+`
+)
+
+func GenerateCfgStr(clusterName string, ip string, ca, token []byte, username string) (string, error) {
```
Comment (shivi28) [inline tenant/pkg/controller/tenantnamespace/tenantnamespace_helper.go]: This is different from VC code

File: tenant/pkg/controller/tenantnamespace/tenantnamespace_helper.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	saList := corev1.ServiceAccountList{}
+	if err := c.List(context.TODO(), &client.ListOptions{}, &saList); err != nil {
+		return "", err
+	}
+
+	//Get secret name of tenant admin service account
+	var saSecretName string
+	for _, eachSA := range saList.Items {
+		for _, each := range eachSA.Secrets {
+			if strings.Contains(each.Name, saName) {
```
Comment (Fei-Guo) [inline tenant/pkg/controller/tenantnamespace/tenantnamespace_helper.go]: You should do more strict check.  First the secret type is type: kubernetes.io/service-account-token and 
the sa name is in its annotation:
annotations:
    kubernetes.io/service-account.name: t1-user1
    kubernetes.io/service-account.uid: e6ed084c-fc32-11e9-8f10-0800278e01e3

**Comment by shivi28** [comment]
Deleting this PR and opened another one https://github.com/kubernetes-sigs/multi-tenancy/pull/569

**Pull request closed without merging**